### ISSUE TITLE
write EPROTO Error

### ISSUE THREAD

**Issue by rainbowhat** [browserless/chrome#575]
write EPROTO 139969703676800:error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol:s23_clnt.c:827:

```
'use strict';

const puppeteer = require('puppeteer');

(async() => {
  // const browser = await puppeteer.launch({ headless: true, args: ['--no-sandbox']});
  try {
    const browser = await puppeteer.connect({
        browserWSEndpoint: 'wss://IPADDRESS:3000/'
      });
    const page = await browser.newPage();
    await page.setViewport({ width: 1024, height: 400 });
    await page.goto('https://google.com/', { timeout: 3000 });
    await page.screenshot({path: 'buddy-screenshot.png'});
    await browser.close();
    
  } catch (err) {
    console.error(err.message);
  }
})();
```

Anyone else facing this issue, i can't seem to connect to ip directly.

**Comment by mikemajara** [comment]
@JohnDotOwl I see you closed this – did you find a solution? Could you summarize it here?
